### DISCUSSION TITLE
Send pictures by mail

### DISCUSSION THREAD

**Discussion by MALAONE1** [jekhokie/raspberry-noaa-v2#354] [Q&A] [answered]
Hello I entered my e-mail in the config and activated it with True.  But there is no mail with pictures.  Do I have to adjust or do anything else?

**Comment by dom-robinson** [comment, marked as answer]
Hiya - did you see this guide: https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/emailing.md 

There is some setup for the mail service in `/home/pi/.msmtprc` that needs completing too ..

**Reply by MALAONE1** [reply]
Thank you for your help this works now.  I still have a question about the font where is in the pictures, such as ground station and satellite name.  Is it possible to make the font larger and to make it from the left to the center

**Reply by dom-robinson** [reply]
Check the https://github.com/jekhokie/raspberry-noaa-v2/blob/master/docs/annotation.md section
